pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -518,11 +518,10 @@ def setup_method(self, method):
         self.offset2 = BDay(2)
 
     def test_different_normalize_equals(self):
-        # equivalent in this special case
-        offset = BDay()
-        offset2 = BDay()
-        offset2.normalize = True
-        assert offset == offset2
+        # GH#???? changed __eq__ to return False when `normalize` doesnt match
+        offset = self._offset()
+        offset2 = self._offset(normalize=True)
+        assert offset != offset2
```

Comment:
After substituting in `self._offset` for `BDay`, this test is identical to 5 others in this file.  I'm putting together a separate PR to try to de-duplicate and parametrize a bunch of things like this.